src/blocks/content-helper/current-post-details/component.tsx

```diff
@@ -225,9 +236,14 @@ function TopReferrersSection( data: PostPerformanceData ) {
 							return null;
 						}
 
+						let referrerUrl = key;
+						if ( key === 'direct' ) {
+							referrerUrl = __( 'Direct', 'wp-parsely' );
+						}
+
 						return (
 							<tr key={ key }>
-								<th title={ key }>{ key }</th>
+								<th title={ referrerUrl }>{ referrerUrl }</th>
 								<td title={
```

Comment:
OK, I can see what's trying to be done, but it's not going to be accessible.

Right now you have a `td` element with a title attribute (not typically read out by screen readers, and touch devices can't access it either since it assumes hovering with a pointer), and inside the cell is an empty div (screen readers won't see anything), followed by a visible number in a new cell.

Instead, what's needed is the `td` (no attribute), followed by the empty div with an `aria-hidden` attribute (so it is definitely ignored by screen readers), and a screen-reader-text span with the percentage, then the number in the next cell.

The resulting HTML of that middle cell would be something like:

```html
<td>
	<div aria-hidden="true" style="..."></div>
	<span class="screen-reader-text">27%</span>
</td>
```

Alternatively, you could use an aria-label instead:

```html
<td>
	<div aria-label="27%" style="..."></div>
</td>
```